crates/forge_main/src/prompt.rs

```diff
@@ -20,7 +18,7 @@ const RIGHT_CHEVRON: &str = "❯";
 #[setters(strip_option, borrow_self)]
 pub struct ForgePrompt {
     pub usage: Option<Usage>,
-    pub mode: Mode,
+    pub agent_id: Option<AgentId>,
```

Comment:
```suggestion
    pub agent_id: AgentId,
```
Lets use `act` as default.